app/src/main/java/io/intrepid/contest/screens/entrysubmission/entryname/EntryNamePresenter.java

```diff
@@ -18,24 +18,17 @@
     @Override
     public void onViewCreated() {
         super.onViewCreated();
-        view.setContestName("Chili Cookoff");
-    }
-
-    @Override
-    public void onEntryNameFocusChanged(boolean isFocused) {
-        if (isFocused) {
-            view.showEntryNameButton();
-        } else {
-            view.hideEntryNameButton();
-        }
+        view.showWelcomeMessage();
```

Comment:
Honestly I wasn't sure if this was something the view could decide on its own!
The contest name used to appear in the welcome message, so it was the presenter's responsibility, but that was removed in the look-and-feel. Not sure if the designers will want it back at some point, but I'm assuming not, so I'll do it in `onCreate`!